Formula/android-sdk.rb

```diff
@@ -31,11 +30,12 @@ class AndroidSdk < Formula
 
   # Version of the android-build-tools the wrapper scripts reference.
   def build_tools_version
-    "23.0.1"
+    "25.0.1"
   end
 
   def install
-    prefix.install "tools", "SDK Readme.txt" => "README"
+    Dir.chdir "../"
+    prefix.install "tools"
```

Comment:
instead of changing directories (by the way `cd "foo"` works, no need for `Dir.chdir`), probably should do this:
```
(prefix/"tools").install buildpath.children - [buildpath/".brew_home"]
```